UNITTESTS/stubs/EmulatedSD.h

```diff
@@ -0,0 +1,77 @@
+#ifndef EMULATEDSD_H
```

Comment:
this file still does not contain a license

Reply:
Thanks. Fixed.